packages/web/components/cards/my-position/status.tsx

```diff
@@ -47,9 +47,11 @@ export const MyPositionStatus: FunctionComponent<
     )
   );
 
+  const rangeDiff = new Dec(Number(upperPrice.sub(lowerPrice).toString()));
```

Comment:
I think this Number conversion is not necessary. You should be able to directly use the Dec objects.

Reply:
k